### PR TITLE
Profile review boxes fixed and popular reviews added

### PR DESCRIPTION
**Pull request by saraf020**
That many additions/deletions are not actually anything I did besides the two sections with the popular and recent reviews. I think my IDE just reformatted it automatically upon opening it. From what I can tell, the "changes" are the same, just with different formatting.

### CODE REVIEW

File: src/app/(Feed)/[profile]/page.tsx
Comment (sloanfinger) [inline src/app/(Feed)/[profile]/page.tsx:1]: Let's also make this layout more responsive for smaller device widths!

File: src/app/(Feed)/[profile]/page.tsx [hunk 1]
```diff
@@ -186,44 +186,58 @@ export default async function ProfilePage({ params }: Props) {
...
+          <div className="mx-auto grid w-full max-w-6xl grid-cols-2 gap-x-4 gap-y-8 rounded-lg bg-zinc-900 px-24 py-8">
+            <h2 className="col-span-full flex items-center gap-3 font-bold uppercase text-amber-400">
+              Recent Reviews
+              <PiRssBold className="-mb-0.5 -ml-0.5"/>
+              <span className="h-[3px] flex-1 bg-current opacity-50"/>
+            </h2>
+
+            <Reviews author={user._id.toHexString()} limit={6} sortBy="recent"/>
+          </div>
+        </section>
```
Comment (sloanfinger) [inline src/app/(Feed)/[profile]/page.tsx:230]: I think we should rework these section layouts:
- Make the section a flex container (in the column direction), and move most of the padding/layout styles from the `div` up.
- Move the `h2` out of the grid, making the `Reviews` component the only child in the grid layout.

File: src/app/(Feed)/[profile]/page.tsx [hunk 2]
```diff
@@ -186,44 +186,58 @@ export default async function ProfilePage({ params }: Props) {
...
+        <section>
+          <div className="mx-auto grid w-full max-w-6xl grid-cols-2 gap-x-4 gap-y-8 rounded-lg bg-zinc-900 px-24 py-8">
+            <h2 className="col-span-full flex items-center gap-3 font-bold uppercase text-amber-400">
+              Popular Reviews
+              <PiRssBold className="-mb-0.5 -ml-0.5"/>
```
Comment (sloanfinger) [inline src/app/(Feed)/[profile]/page.tsx:235]: Maybe look for a more suitable icon for this section? I'd prefer we use something from the same set ([Phosphor Icons](https://react-icons.github.io/react-icons/icons/pi/)) and keep the bold weight.

**Review by sloanfinger — CHANGES_REQUESTED** [review]

**Comment by saraf020** [comment]
ESlint warning and profile review width responsiveness should be fixed

**Review by sloanfinger — APPROVED** [review]

**Pull request merged**